setup.py

```diff
@@ -28,7 +28,7 @@
         "jsonpointer>1.13",
         "rfc3987",
         "strict-rfc3339",
-        "uritemplate>3.0.0",
+        "uritemplate>=3.0.0",
```

Comment:
I know that the > was failing here, but that's because the patch with this wasn't merged  yet (and in fact 3.0.0 doesn't have what we need).

So I think it was correct as-is [and probably even needs some more work once the upstream PR is merged].

Maybe you can chime in upstream with any comments? The PR is https://github.com/python-hyper/uritemplate/pull/36